src/EventTickets/ListTable/Columns/SalesAmountColumn.php

```diff
@@ -46,22 +47,26 @@ public function getCellValue($model, $locale = ''): string
     {
         $ticketTypes = [];
         foreach ($model->ticketTypes()->getAll() ?? [] as $ticketType) {
+            $salesCount = $ticketType->eventTickets()->count();
+            $ticketsAvailable = $ticketType->capacity - $salesCount;
+
             $ticketTypes[$ticketType->id] = [
                 'price' => $ticketType->price,
                 'capacity' => $ticketType->capacity,
+                'ticketsAvailable' => $ticketsAvailable,
             ];
         }
 
         $salesTotal = array_reduce(
```

Comment:
As an observation, this would be a great use case for a `Collection` class.

```php
( new TicketCollection($eventTickets) )->sumAmount();
```

We don't currently have collections in the framework, but if we did it would be a great way to make this sort of logic more declarative.